<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POSModeWidgetClass</class>
 <widget class="QWidget" name="POSModeWidgetClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POSModeWidget</string>
  </property>
  <widget class="QLabel" name="subtotalCountText">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>630</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>630</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtotal: $</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>541</width>
     <height>761</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="buttonGrid"/>
  </widget>
  <widget class="QTableWidget" name="itemTable">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>0</y>
     <width>256</width>
     <height>631</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="newOrderButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <fontweight>ExtraBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Start New Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmOrderButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>710</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order being prepped:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="orderDropdown">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="indexSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteIndexButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Item Number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePreviousItemButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Previous Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalCountText">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>650</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>650</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total: $</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPOSModeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit POS Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>760</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInputBox">
   <property name="geometry">
    <rect>
     <x>872</x>
     <y>680</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>680</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
